### PR TITLE
Changed App logo

### PR DESCRIPTION
**Pull request by linjiang82**
**Problem:**
Changed App Icon
**Solution:**

### CODE REVIEW

**Comment by KipCrossing** [comment]
@connorhsm do you remember how to change the logo/icon background for android?

**Comment by connorhsm** [comment]
I'm assuming the Android icon has not updated as expected, where as the iOS one has?

File: pubspec.yaml [hunk 1]
```diff
@@ -75,9 +75,10 @@ dev_dependencies:
 flutter_icons:
   android: true
   ios: true
-  image_path: "assets/graphics/static-logo.png"
+  image_path: "assets/graphics/static-logo-no-name.png"
+  image_path_android: "assets/graphics/static-logo-no-name-android.png"
   adaptive_icon_background: "#FDF7B9"
```
Comment (connorhsm) [inline pubspec.yaml:80]: This hex colour value needs to be updated to the new background colour.

**Review by connorhsm — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (linjiang82) [reply pubspec.yaml:80]: change to Black as the background color in Ios?

Comment (connorhsm) [reply pubspec.yaml:80]: You should use colours as set in the design doc. The iOS image you've used will also need to be changed.
You can find the colour palette [here](https://github.com/voteflux/voting_app/issues/142#issuecomment-691466612)
The darkest colour would be best (#1C1F27)

**Comment by KipCrossing** [comment]
Can you also merge the latest master into this fork?

Also, you can create branches - so no need to fork anymore

File: pubspec.yaml (hunk 1 above)
Comment (linjiang82) [reply pubspec.yaml:80]: Ok, I will modify it shortly.

File: android/app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+<<<<<<< HEAD
```
Comment (connorhsm) [inline android/app/src/main/res/values/colors.xml:3]: You have an unresolved merge conflict

Comment (connorhsm) [reply android/app/src/main/res/values/colors.xml:3]: Is this change relevant for the app icon?

File: lib/ui/views/startup_view.dart [hunk 3]
```diff
@@ -40,7 +40,7 @@ class StartupView extends HookWidget {
 class StartupViewModel extends ChangeNotifier {
   final log = Logger();
   bool _animationComplete = false;
-  String _destinationRoute = Routes.profilePage;
+  String _destinationRoute = Routes.signin;
```
Comment (connorhsm) [inline lib/ui/views/startup_view.dart:43]: Is this change relevant for the app icon?

Comment (linjiang82) [reply lib/ui/views/startup_view.dart:43]: No, this was for testing on the signup page, I did not commit it last time and accidentally added it here

**Comment by linjiang82** [comment]
Close this PR as I have created a new one

**Pull request closed without merging**